app/namespace.go

```diff
@@ -829,6 +829,63 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				},
 			},
 		},
+		{
+			Name:    "update-codec-server",
+			Usage:   "Update codec server config used to decode encoded payloads through remote endpoint",
+			Aliases: []string{"ucs"},
+			Flags: []cli.Flag{
+				NamespaceFlag,
+				&cli.StringFlag{
+					Name:     "endpoint",
+					Usage:    "The codec server endpoint to decode payloads for all users interacting with this Namespace",
```

Comment:
IIUC this must be in https form.  Will you please ensure that the description mentions https is required?